src/components/Searchbar.tsx

```diff
@@ -203,6 +213,8 @@ class Searchbar extends React.Component<Props> {
           placeholderTextColor={colors.placeholder}
           selectionColor={colors.primary}
           underlineColorAndroid="transparent"
+          multiline={multiline || false}
+          numberOfLines={numberOfLines}
```

Comment:
Can you default this to 1 so it doesn't appear as `undefined` ?

Reply:
fixed